### ISSUE TITLE
cannot run e2e tests locally

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#782]
### Steps to Reproduce the Problem

- modify e2e/Dockerfile to test localhost
- docker build -t permaplant-e2e:build ./e2e
- docker run -it permaplant-e2e:build
- make test-e2e
- python3  -m pytest -n auto e2e/ --retries 2 --video retain-on-failure --html=report.html --self-contained-html --cucumberjson=cucumber.json

### Expected Result

successfully executed e2e tests

### Actual Result

```
root@b25b26e9d5d9:/home/batman/permaplante2e# make test-e2e
make: *** No rule to make target 'test-e2e'.  Stop.
root@b25b26e9d5d9:/home/batman/permaplante2e# python3  -m pytest -n auto e2e/ --retries 2 --video retain-on-failure --html=report.html --self-contained-html --cucumberjson=cucumber.json
ImportError while loading conftest '/home/batman/permaplante2e/conftest.py'.
conftest.py:4: in <module>
    from e2e.pages.home import HomePage
E   ModuleNotFoundError: No module named 'e2e'
```

### PermaplanT version

master

### Operating System

Linux

### Browser

chromium

**Comment by markus2330** [comment]
I just noticed that `make run-storybook` installs e2e deps?

**Comment by 4ydan** [comment]
Yea, we should split installation now. At the beginning everything was added to install, but I agree, this doesnt make any sense.

**Comment by 4ydan** [comment]
Thank you btw for reporting this. Whilest developing the e2e tests, I was only using that Dockerfile inside the pipeline and it seems like Jenkins does the paths a bit different when building images. Jenkins copies the whole root folder, thats why we can use `make test-e2e` inside the pipeline. When building the image locally though, you only have `/e2e/` and you are already inside that folder.

In #784 the following syntax now to run that docker image.

```sh
in /workspaces/PermaplanT/e2e $ docker build -t permaplant-e2e .
in /workspaces/PermaplanT/e2e $ docker run --network="host" permaplant-e2e ./e2e.sh
```
`--network="host"` only needed if your app is running on localhost.
I hope you can run e2e tests now locally 🚀.

> I just noticed that make run-storybook installs e2e deps?

Thank you. Should also be fixed in #784
